webdriver/tests/bidi/browsing_context/navigation_started/navigation_started.py

```diff
@@ -93,7 +93,7 @@ async def test_iframe(
 ):
     events = []
 
-    async def on_event(method, data):
+    async def on_event(_, data):
```

Comment:
keeping method looks better to me

Reply:
We made similar changes (=removing unused variables) in the past. Also, it keeps the linters happier in general.

Do you think it looks better because of discoverability / documentation, or actual readability?

Reply:
Everything in the PR looks fine to me, but we might need to discuss this.

I think that this type of linter rule is not a great fit for callbacks, where you don't have any flexibility to change the arguments. Which linter/linter-rule is this, is this something you could disable by configuration until we get more feedback from others?

Reply:
Reverted.

Re: which linter rule: it's something that is in my VSCode set-up, possibly https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance. I'll see how to disable it on my end.